style.css

```diff
@@ -2,42 +2,267 @@
 
 @import url(./components/components.css);
 
-*{
-    padding: 0;
-    margin: 0;
-    box-sizing: border-box;
+html {
+    scroll-behavior: smooth;
 }
 
-:root{
-    font-size: 62.5%;
-}
-
-.container{
+.container {
     max-width: 80rem;
-    margin:auto;
+    margin: auto;
     padding: 1.6rem;
     min-height: 100vh;
     background-color: #eee;
```

Comment:
use var for colour.

Reply:
oh that container class was extra code will remove